mllib/src/main/scala/org/apache/spark/mllib/regression/impl/GLMRegressionModel.scala

```diff
@@ -23,7 +23,7 @@ import org.json4s.jackson.JsonMethods._
 import org.apache.spark.SparkContext
 import org.apache.spark.mllib.linalg.Vector
 import org.apache.spark.mllib.util.Loader
-import org.apache.spark.sql.{DataFrame, Row, SQLContext}
+import org.apache.spark.sql.{DataFrame, Dataset, Row, SQLContext}
```

Comment:
I'm surprised you need this change without any other changes in the file? How could that be?